packages/tests/tests/light-push/multiple_pubsub.node.spec.ts

```diff
@@ -60,19 +61,37 @@ describe("Waku Light Push (Autosharding): Multiple Shards", function () {
     await teardownNodesWithRedundancy(serviceNodes, waku);
   });
 
-  it("Subscribe and receive messages on 2 different pubsubtopics", async function () {
+  it("Subscribe and receive messages on 2 different pubsubtopics with v3 protocol", async function () {
```

Comment:
I would expect all test now use v3.

And maybe have one test that run a nwaku node that only support v2 to confirm that compatibility remains.

Reply:
since we support `v2` and `v3` at the same time we have to have comprehensive test suit for both until we remove `v2`

for that I would advice to run the same times for both for now

Reply:
and we need to cover additional test case were we run:
- only for v2 - exists already
- only for v3 - should re-use existing
- one node v2 and one v3 - should be created 

discussed at PM @adklempner